New package: HybridDynamicModels v0.0.5

JuliaRegistrator opened the pull request:
- Registering package: HybridDynamicModels
- Repository: https://github.com/vboussange/HybridDynamicModels.jl
- Created by: @vboussange
- Version: v0.0.5
- Commit: e35503102b8628944088f1ca8822c89433d8da67
- Reviewed by: @vboussange
- Reference: https://github.com/vboussange/HybridDynamicModels.jl/issues/2#issuecomment-3322494541
- Description: Lux layers and utilities to build and train hybrid dynamic models.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835981dadf9f5e4417874c4a157eb60e4d548c848112ff393b1801359930664888148c4a510d5fa90cf90ede8df2a98ba4bfb77897797ed373987dabe183eb6e854600978e485c6a1ba3aaefa4166268ed593c0cb87083565464402ff4f2934695faeda7f8e186ad4a3dec416c6c164e3690181072508df81c474181c92f27c9d80c4 -->

vboussange commented:
@JuliaRegistrator register

vboussange commented:
@JuliaRegistrator register()

[noblock]

goerz commented:
See [I commented @JuliaRegistrator register on a pull request in the General registry, but nothing happened.](https://github.com/JuliaRegistries/General?tab=readme-ov-file#i-commented-juliaregistrator-register-on-a-pull-request-in-the-general-registry-but-nothing-happened)

[noblock]

goerz commented:
Yeah, it's a bit confusing. What it should say is that you can retrigger a registration the same way you triggered the original registration. In your case, filling out the web-form on JuliaHub again.

> I opened an issue to post a comment on the package repo (https://github.com/vboussange/HybridDynamicModels.jl/issues/2), but it did not seem to do anything.

That's very strange! It might be a permissions issue... Did you add the bot your repository, according to the ["Install Registrator" instructions](https://github.com/JuliaRegistries/Registrator.jl/?tab=readme-ov-file#install-registrator)?

I'm never totally sure how registration via an issue is supposed to work in full detail. I know it's something people commonly due, but it's not actually a documented method in the [usage instructions](https://github.com/JuliaRegistries/Registrator.jl/?tab=readme-ov-file#how-to-use). What I do is comment on the commit... as I just saw you tried as well, in https://github.com/vboussange/HybridDynamicModels.jl/commit/e35503102b8628944088f1ca8822c89433d8da67#commitcomment-166286128

Since that didn't also didn't get a reply, my best guess is still that you didn't give permissions to the bot.

[noblock]

JuliaRegistrator committed:
New package: HybridDynamicModels v0.0.5

UUID: 7e133797-8c8e-49d9-acf3-b153daa40e06
Repo: https://github.com/vboussange/HybridDynamicModels.jl.git
Tree: a518cb7b1f292b8399f389588f31757b5316705f

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
